src/sentry_plugins/asana/plugin.py

```diff
@@ -202,7 +203,14 @@ def get_config(self, *args, **kwargs):
             client = self.get_client(user)
         except PluginIdentityRequired as e:
             self.raise_error(e)
-        workspaces = client.get_workspaces()
+        try:
+            workspaces = client.get_workspaces()
+        except Exception as e:
+            if (
+                e.response.reason == "Bad Request"
+                and e.response.url == "https://app.asana.com/-/oauth_token"
+            ):
+                raise PluginIdentityRequired(ERR_BEARER_EXPIRED)
```

Comment:
@ceorourke What about other errors? Shouldn't we re-raise anything that doesn't meet this condition? Pretty sure this is going to cause problems for things like timeouts.